be/src/exec/analytor.cpp

```diff
@@ -215,7 +215,13 @@ Status Analytor::prepare(RuntimeState* state, ObjectPool* pool, RuntimeProfile*
                 real_fn_name += "_in";
                 _need_partition_materializing = true;
             }
-            func = get_window_function(real_fn_name, arg_type.type, return_type.type, is_input_nullable, fn.binary_type,
+            const auto& fname = fn.name.function_name;
+            auto real_arg_type = arg_type.type;
```

Comment:
how about using function version?

Reply:
Here is a ancient bug,  max_by/min_by's specialization type is determined by the second parameter,  while in analytical operator, misuse the first type.

By the way, function version seems not a good choice,  FE directly choose the agg name of real implementation seems a simple way.